packages/cozy-harvest-lib/src/components/hooks/useAppLinkWithStoreFallback.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { Q, models } from 'cozy-client'
+
+const { applications: ApplicationsModel } = models
+
+const useAppLinkWithStoreFallback = (slug, client, path = '') => {
+  const [fetchStatus, setFetchStatus] = useState('loading')
+  const [isInstalled, setIsInstalled] = useState(true)
+  const [url, setURL] = useState()
+
+  useEffect(() => {
+    const load = async () => {
+      try {
+        const apps = await client.query(Q('io.cozy.apps'))
```

Comment:
Why do we load all the apps when we need only one ?
Could we use a fetch policy here to limit the number of calls and limit the spinning hell ?

Reply:
Can we do that? I mean having a fetchPolicy on a query()? I thought we were waiting a useQuery() hook or something to do so

Reply:
We can't at the moment, see https://github.com/cozy/cozy-client/issues/646